scripts/bootstrapOrUpdate.ts

```diff
@@ -21,8 +54,8 @@ async function bootstrapOrUpdate() {
   const collection = database.collection("Collections")
 
   try {
-    if (connection.isConnected) {
-      for (const entry of data) {
+    if (connection.isConnected && formattedCollections) {
```

Comment:
instead of relying on the global `formattedCollections`, make `bootstrapOrUpdate` take in an argument for `data`